{
  "uid" : "56e9c8909dd4f048",
  "name" : "CreditAPIValidateResponseWithInvalidToken",
  "fullName" : "testCases.IqSoft_API_4_Credit_Negative_Test.CreditAPIValidateResponseWithInvalidToken",
  "historyId" : "216d89b1b4125e437fb41555d2e3ee82",
  "time" : {
    "start" : 1666984247607,
    "stop" : 1666984248173,
    "duration" : 566
  },
  "description" : "Verify Credit API_s response with invalid Token",
  "descriptionHtml" : "<p>Verify Credit API_s response with invalid Token</p>\n",
  "status" : "failed",
  "statusMessage" : "The following asserts failed:\n\tResponseCode: 0 expected [22] but found [0],\n\tError Description:  expected [error login] but found [],\n\texpected [1041.21] but found [1042.21]",
  "statusTrace" : "java.lang.AssertionError: The following asserts failed:\n\tResponseCode: 0 expected [22] but found [0],\n\tError Description:  expected [error login] but found [],\n\texpected [1041.21] but found [1042.21]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat testCases.IqSoft_API_4_Credit_Negative_Test.CreditAPIValidateResponseWithInvalidToken(IqSoft_API_4_Credit_Negative_Test.java:101)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:806)\r\n\tat org.testng.TestRunner.run(TestRunner.java:601)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:433)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "setupSuite",
    "time" : {
      "start" : 1666984239704,
      "stop" : 1666984239738,
      "duration" : 34
    },
    "description" : "",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "testStage" : {
    "description" : "Verify Credit API_s response with invalid Token",
    "status" : "failed",
    "statusMessage" : "The following asserts failed:\n\tResponseCode: 0 expected [22] but found [0],\n\tError Description:  expected [error login] but found [],\n\texpected [1041.21] but found [1042.21]",
    "statusTrace" : "java.lang.AssertionError: The following asserts failed:\n\tResponseCode: 0 expected [22] but found [0],\n\tError Description:  expected [error login] but found [],\n\texpected [1041.21] but found [1042.21]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat testCases.IqSoft_API_4_Credit_Negative_Test.CreditAPIValidateResponseWithInvalidToken(IqSoft_API_4_Credit_Negative_Test.java:101)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:806)\r\n\tat org.testng.TestRunner.run(TestRunner.java:601)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:433)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "75b2c320f2f88104",
      "name" : "GetBalanceAPI:  Url:  https://gobet.tn/iapi/iq/GetBalance        RequestBody",
      "source" : "75b2c320f2f88104.txt",
      "type" : "text/plain",
      "size" : 50
    }, {
      "uid" : "4728f4b29d96415e",
      "name" : "GetBalanceAPI:  ResponseTime  168ms        ResponseBody",
      "source" : "4728f4b29d96415e.txt",
      "type" : "text/plain",
      "size" : 83
    }, {
      "uid" : "eac38ca0935a6555",
      "name" : "CreditAPI:  Url:  https://gobet.tn/iapi/iq/Credit        RequestBody",
      "source" : "eac38ca0935a6555.txt",
      "type" : "text/plain",
      "size" : 161
    }, {
      "uid" : "ca23707aedaeb803",
      "name" : "CreditAPI:  ResponseTime  194ms        ResponseBody",
      "source" : "ca23707aedaeb803.txt",
      "type" : "text/plain",
      "size" : 218
    }, {
      "uid" : "593470ea415cd2f5",
      "name" : "GetBalanceAPI:  Url:  https://gobet.tn/iapi/iq/GetBalance        RequestBody",
      "source" : "593470ea415cd2f5.txt",
      "type" : "text/plain",
      "size" : 50
    }, {
      "uid" : "cac5626a0c962647",
      "name" : "GetBalanceAPI:  ResponseTime  183ms        ResponseBody",
      "source" : "cac5626a0c962647.txt",
      "type" : "text/plain",
      "size" : 83
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 6,
    "shouldDisplayMessage" : true,
    "hasContent" : true,
    "stepsCount" : 0
  },
  "afterStages" : [ {
    "name" : "tearDownSuite",
    "time" : {
      "start" : 1666984267260,
      "stop" : 1666984267260,
      "duration" : 0
    },
    "description" : "",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "labels" : [ {
    "name" : "package",
    "value" : "testCases.IqSoft_API_4_Credit_Negative_Test"
  }, {
    "name" : "testClass",
    "value" : "testCases.IqSoft_API_4_Credit_Negative_Test"
  }, {
    "name" : "testMethod",
    "value" : "CreditAPIValidateResponseWithInvalidToken"
  }, {
    "name" : "parentSuite",
    "value" : "IQsoft API Integration"
  }, {
    "name" : "suite",
    "value" : "API Integration Automation Test"
  }, {
    "name" : "subSuite",
    "value" : "testCases.IqSoft_API_4_Credit_Negative_Test"
  }, {
    "name" : "host",
    "value" : "DESKTOP-52C1ANV"
  }, {
    "name" : "thread",
    "value" : "21028@DESKTOP-52C1ANV.main(1)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "severity",
    "value" : "blocker"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "blocker",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "56e9c8909dd4f048.json",
  "parameterValues" : [ ]
}